pages/changelog.js

```diff
@@ -114,17 +114,15 @@ export default function Changelog() {
                     </div>
                   </div>
                   <div className="mt-2 sm:flex sm:justify-between">
-                    <div className="sm:flex dark:text-primary-medium-low">
+                    <div className="sm:flex dark:text-primary-low">
                       {change.description}
                     </div>
-                    <div className="mt-2 flex items-center text-sm text-primary-medium sm:mt-0">
+                    <div className="flex items-center mt-2 text-sm text-primary-medium sm:mt-0">
```

Comment:
I think this change is not required?

```suggestion
                    <div className="mt-2 flex items-center text-sm text-primary-medium sm:mt-0">
```